Client/Frontend/Login Management/LoginListTableViewCell.swift

```diff
@@ -8,61 +8,60 @@ class LoginListTableViewSettingsCell: ThemedTableViewCell {
     override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
         super.init(style: style, reuseIdentifier: reuseIdentifier)
     }
-    
+
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 }
 
 class LoginListTableViewCell: ThemedTableViewCell {
     private let breachAlertSize: CGFloat = 24
-    lazy var breachAlertImageView: UIImageView = {
-        let imageView = UIImageView(image: BreachAlertsManager.icon)
-        imageView.isHidden = true
-        return imageView
-    }()
-    lazy var breachAlertContainer: UIView = {
-        let view = UIView()
-        view.addSubview(breachAlertImageView)
+    lazy var breachAlertImageView: UIImageView = .build { imageView in
+        imageView.image = BreachAlertsManager.icon
```

Comment:
This image needs to stay hidden by default with `imageView.isHidden = true`. Breaches are currently handled by setting this to `false` in the LoginListViewController when there's actually a breach that was detected. If this is not hidden by default the breach icon appears by default which is not what we want.

![Screen Shot 2022-03-29 at 6 59 38 PM](https://user-images.githubusercontent.com/11338480/160720246-cc0f86f2-193e-41b0-9223-213cc130d3eb.png)